public/js/bundle.js

```diff
@@ -94,6 +113,7 @@ class TableView {
   initDomReferences() {
     this.headerRowEl = document.querySelector('THEAD TR');
     this.sheetBodyEl = document.querySelector('TBODY');
+    this.sheetSumEl = document.querySelector('TFOOT');
```

Comment:
Consistency in codebase is very important and makes for more readable code. "sheetSumEl" is fine, but since what you're making is a table footer element, even better if it looks like our conventions for the header element, which we called "headerRowEl".